src/main/java/org/prnhs/javaee/swim/core/CoreConfig.java

```diff
@@ -0,0 +1,14 @@
+package org.prnhs.javaee.swim.core;
```

Comment:
our configuration class that instructs the system how the core is configured.  We placed it in the base core package, but it also may appropriately live in core.config package if we have more than one config associated with core related obligations.